lib/ansible/modules/net_tools/dnsimple.py

```diff
@@ -95,7 +92,7 @@
     state: absent
   delegate_to: localhost
 
-# create a test.my.com A record to point to 127.0.0.01
+# create a test.my.com A record to point to 127.0.0.1
```

Comment:
Maybe use `- name: create a test.my.com A record to point to 127.0.0.1` instead of putting the information in a comment?